lumina-node-shim/worker.js

```diff
@@ -0,0 +1,24 @@
+Error.stackTraceLimit = 99;
```

Comment:
as we anyway do a glueing npm package, can we split this into `index.js` and `worker.js`?

Reply:
What exactly would be the split? 
The worker code (inside the if) would be in worker, but I think the function starting worker should land there too, to ensure paths not going out of sync on any file moves.

Reply:
in `index.js` the `spawnNode` and `export * from "lumina-node-wasm";`.

Worker script can then drop whole `if` because it's just a hack to re-use the module:
```js
import init, { NodeWorker } from "lumina-node-wasm"

Error.stackTraceLimit = 99;

init().then(async () => {
  let worker = new NodeWorker();
  console.log("starting worker: ", worker);
  worker.connect(self)

  while (true) {
    await worker.poll();
  }
});
```

Reply:
I think it's unlikely the path will get invalidated as this shim would then just have only `index.js` and `worker.js`

Reply:
Nice, two birds with once stone :rocket: split the script